lib/pub_sub/poller.rb

```diff
@@ -1,31 +1,38 @@
 module PubSub
   class Poller
-    def initialize(queue_url, verbose = false)
-      @queue_url = queue_url
+    def initialize(verbose = false)
       @verbose = verbose
     end
 
     def poll
-      poller.poll(config) do |message|
-        if @verbose
-          PubSub.logger.info(
-            "PubSub received: #{message.message_id} - #{message.body}"
-          )
+      loop do
+        Breaker.current_breaker.run do
+          poller.poll(idle_timeout: 60) do |message|
```

Comment:
Are we no longer using a custom `visibility_timeout` value, and instead just relying on the `VisibilityTimeout` of the queue itself?  If this is the case, we should remove the `visibility_timeout` attribute from [configuration.rb](https://github.com/mastfish/pub_sub/blob/multi-region-failover-1441079293/lib/pub_sub/configuration.rb#L12)